build.sh

```diff
@@ -242,7 +241,7 @@ if [ "$DO_UPDATE" == true ] || [ ! -f $HTML_CACHE/caniuse.json ]; then
   rm -f $HTML_CACHE/caniuse.json
   $QUIET || echo "Downloading caniuse data..."
   curl $($VERBOSE || echo "-s") \
-    -o $HTML_CACHE/caniuse.json -k \
```

Comment:
Last I checked this was required, but if you've tested it and it works, then cool, seems like a good idea.